airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/split_secrets/JsonSecretsProcessor.java

```diff
@@ -27,10 +27,7 @@
 public class JsonSecretsProcessor {
 
   @Builder.Default
-  final private boolean maskSecrets = true;
```

Comment:
@benmoriceau this was only used for the export, right? that was the only usage I could see, but lmk if you think there might be others.

Reply:
> @benmoriceau this was only used for the export, right? that was the only usage I could see, but lmk if you think there might be others.

I need to double check. I remember a PR from Pedro using it

Reply:
Yeah, it is ok to remove. Masking the secret should be the default.